notebooks/09-Importing-Data.ipynb

```diff
@@ -0,0 +1,1706 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4cd98d02-ae84-4af9-b6c4-01deb8d6f361",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    },
+    "tags": []
+   },
+   "source": [
+    "# Importing Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d6cda866-4815-4728-a1b0-9eccc0fb383f",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    },
+    "tags": []
+   },
+   "source": [
+    "This lesson introduces you to importing tabular data with pandas. It also illustrates how you can interact with relational databases via SQL, along with how to import common non-tabular data such as JSON and pickle files. This comes prior to the tutorial on usage of pandas, but you will not need to understand pandas to follow along here.\n",
+    "\n",
+    "## Learning Objectives\n",
+    "\n",
+    "1. Import tabular data with Pandas.\n",
+    "2. Assess DataFrame attributes and methods.\n",
+    "3. Import alternative data files such as SQL tables, JSON, and pickle files."
```

Comment:
Is the full paragraph needed? Outside of the note about later pandas content, it seems redundant to the list.

Reply:
+1

Reply:
I've removed the paragraph in favor of the bullet points.